dpnp/dpnp_utils/dpnp_utils_pad.py

```diff
@@ -642,8 +642,7 @@ def dpnp_pad(array, pad_width, mode="constant", **kwargs):
 
             # compute indices for the iteration axes, and append a trailing
             # ellipsis to prevent 0d arrays decaying to scalars
-            # TODO: replace with dpnp.ndindex when implemented
```

Comment:
There is another TODO for replacing `numpy.ndindex` with `dpnp.ndindex` in 
`dpnp_iface_functional.py`.
https://github.com/IntelPython/dpnp/blob/master/dpnp/dpnp_iface_functional.py#L154